rust/sbp/src/lib.rs

```diff
@@ -169,6 +169,175 @@ pub enum FramerError {
     NoSenderId,
 }
 
+#[cfg(feature = "swiftnav-rs")]
+mod swiftnav_rs_conversions {
```

Comment:
pr to remove them in swiftnav-rs here - https://github.com/swift-nav/swiftnav-rs/pull/50